odl/space/numpy_ntuples.py

```diff
@@ -563,7 +587,7 @@ class Fn(FnBase, Ntuples):
     Its elements are represented as instances of the `FnVector` class.
     """
 
-    def __init__(self, size, dtype, **kwargs):
+    def __init__(self, size, dtype='float64', **kwargs):
```

Comment:
Not exactly. The `default_dtype()` knew about the field (before you made it optional :-)) so it knew if a real or complex space was desired, and there were the canonical defaults. Before that, there is none IMO.